### PR TITLE
[NeuralChat] add optimized SadTalker to Video plugin in NeuralChat

### PR DESCRIPTION
**Pull request by Spycsh**
## Type of Change

feature

## Description

We plan to enable Video plugin, mainly with SadTalker optimization, in following aspects

- [x] implement SadTalker and instance parallelism bf16 optimziation and integrate it into Video plugin in NeuralChat
- [x] expose restful api to frontend to call image+audio => video functionalities
- [x] offer pipeline example to call image+audio => video & image+text => video (with audio plugin)
- [x] code refactoring
- [x] pass UT

JIRA ticket: [NLPTOOLKIU-974](https://jira.devtools.intel.com/browse/NLPTOOLKIU-974)

## Expected Behavior & Potential Risk

Add video plugin providing face animation

## How has this PR been tested?

UT already added

## Dependency Change?

in the requirements.txt

### CODE REVIEW

**Comment by Spycsh** [comment]
I'll try to refactor the code. Now it contains so much redundancy.

File: intel_extension_for_transformers/neural_chat/pipeline/plugins/video/face_animation/src/audio2exp_models/networks.py
Comment (lvliang-intel) [inline intel_extension_for_transformers/neural_chat/pipeline/plugins/video/face_animation/src/audio2exp_models/networks.py:63]: Why add this absolute checkpoint path?

Comment (Spycsh) [reply intel_extension_for_transformers/neural_chat/pipeline/plugins/video/face_animation/src/audio2exp_models/networks.py:63]: Thanks for pointing out! It's a redundant path.

**Review by lvliang-intel — APPROVED** [review]

**Comment by VincyZhang** [comment]
@Spycsh The added UT not run in CI, did you have local test result?

**Comment by hshen14** [comment]
> @Spycsh The added UT not run in CI, did you have local test result?

We should enable UT to track the functionality though we don't track the coverage seriously for SadTalker.

**Comment by VincyZhang** [comment]
> > @Spycsh The added UT not run in CI, did you have local test result?
> 
> We should enable UT to track the functionality though we don't track the coverage seriously for SadTalker.

The added UT is disabled in CI, so this PR did not trigger this UT.  We can track all the UT in nightly test. But before this PR merge, we need to know if this UT is working.

**Comment by Spycsh** [comment]
@VincyZhang , @hshen14 , I will fix this today.

**Comment by chensuyue** [comment]
depend on https://github.com/intel/intel-extension-for-transformers/pull/706

**Comment by chensuyue** [comment]
CI UT pass, coverage regression is reasonable cause we test the new code in nightly UT.   
Nightly UT failed, please check the issue: https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-neuralchat-ut/46/artifact/unit_test_neuralchat_3.9_cpu.log

**Comment by chensuyue** [comment]
[UT test](https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-neuralchat-ut/54/artifact/unit_test_neuralchat_3.9_cpu.log) pass with the new added case, can be merged after CI pass.

**Review by chensuyue — APPROVED** [review]

**Pull request merged**
